setup.py

```diff
@@ -7,7 +7,7 @@
     ],
     'torch': ['torch~=1.2'],
     'jax': ['jax~=0.1,>0.1.51', 'jaxlib~=0.1,>0.1.33'],
-    'xmlio': ['uproot'],
+    'xmlio': ['uproot~=3.6'],  # Future proof against uproot4 API changes
```

Comment:
I think it should be `'uproot<4.0'`. I don't plan on anything other than bug-fixes for the Uproot 3 branch, but in principle there could be an Uproot 3.7.x. The space under 4.0 is squeezed, though: Uproot 3-style API will never be named "4" (or higher).

Reply:
@jpivarski  Hm, I guess I'm missing something. This accomplishes the same thing while establishing a minimum `uproot` release (of `v3.6`), as the version of `uproot` that is installed in CI for this PR is 

```
uproot                 3.11.7
uproot-methods         0.7.4
```

Reply:
I didn't realize that's what the syntax meant. Okay!

Reply:
Yup! I think I only learned about the it last year (c.f. PR #424) &mdash; super handy!